README.md

```diff
@@ -0,0 +1,33 @@
+# vpphelper
+
+vpphelper provides helper functions (using govpp) that helps to connect to VPP and configure VPP using a configuration (.conf) file.
+
+* There is an `option` struct defined that has two fields:  
+	1. `rootDir` : string having path to root directory
+	2. `vppConfig` : multiline string having contents of a VPP configuration (.conf) file. Here, `%[1]s` needs to be used wherever you want to dynamically insert root directory path.
+		
+		Example: 
+		```go
+		`unix { log %[1]s/var/log/vpp/vpp.log }` 
+		```
+* `StartAndDialContext` function starts VPP using a VPP configuration file and returns a `Connection` to VPP. 
+
+	1. If you want to start VPP with default values for `option` struct
```

Comment:
Changing constants isn't really a reasonable way for users to use a library... just document the 'With*' functions :)

This [Options pattern](https://www.sohamkamani.com/golang/options-pattern/) used here is a pretty common Go idiom :)